tachyon/base/strings/string_util.h

```diff
@@ -146,11 +146,11 @@ std::string VectorToString(const std::vector<T>& data) {
 }
 
 template <typename T>
-std::string Vector2DToString(const std::vector<std::vector<T>>& data) {
+std::string Container2DToString(const std::vector<std::vector<T>>& data) {
```

Comment:
Should `T` be changed to `container` and this function intake a container of `containers`?